elements/rh-cta/rh-cta.css

```diff
@@ -0,0 +1,318 @@
+:host {
+  display: inline-flex;
+  position: relative;
+  z-index: 0;
+  align-items: center;
+  max-width: max-content;
+  background-color: var(--rh-cta-background-color);
+  border-radius: var(--rh-border-radius-default, 0.1875rem);
+  border: var(--rh-border-width-md, 0.125rem) solid var(--rh-cta-border-color, transparent);
+  cursor: pointer;
+  --rh-context-background-color: var(--rh-cta-background-color) !important;
+}
+
+::slotted(*) {
+  white-space: normal;
+  display: inline;
+  padding: 0.6rem, 0 !important;
+  color: inherit !important;
+  font-family: inherit !important;
+  font-size: inherit !important;
+  font-weight: inherit !important;
+  line-height: inherit !important;
+  text-decoration: inherit !important;
+}
+
+::slotted(button),
+::slotted(input) {
+  background-color: transparent;
+  border: medium none;
+  margin: 0px;
+  padding: 0px;
+  text-align: left;
+}
+
+#container {
+  display: block;
+  white-space: nowrap;
+  min-width: 100%;
+  color: var(--rh-cta-color);
+  font-family: var(--rh-font-family-heading, "RedHatDisplay", "Overpass", Helvetica, Arial, sans-serif);
```

Comment:
see https://github.com/RedHat-UX/red-hat-design-tokens/pull/18
TL;DR: it might end up being `RedHatDisplay, "Red Hat Display"`, etc.
I'd like to write some automation that would update our components' css whenever we update the tokens, but we could do this manually as well